CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changes
 
 - LsiModel: Only log top words that actually exist in the dictionary (PR [#3091](https://github.com/RaRe-Technologies/gensim/pull/3091), [@kmurphy4](https://github.com/kmurphy4))
 - [#3115](https://github.com/RaRe-Technologies/gensim/pull/3115): Make LSI dispatcher CLI param for number of jobs optional, by [@robguinness](https://github.com/robguinness))
+- fix bug when loading saved Phrases model (PR [#3116](https://github.com/RaRe-Technologies/gensim/pull/3116), [@aloknayak29](https://github.com/aloknayak29))
```

Comment:
Don't we have a script that spits out a formatted list of all PRs and tickets automatically, during a release?

Reply:
We do, but I've been keeping track of them as we merge them in smart_open, and I've found that way to be more manageable (less risk of missing PRs), so I'm trying out the same approach with gensim.

Reply:
OK, no problem. One advantage of the script is consistency in formatting – see for example the change here :)

Reply:
Yeah, I've got a script that handles these changes, but multiple versions of it were floating around my work/home machines. It's now committed to the repo, so once we nail down the format of the entries, the formatting should no longer be an issue.

Reply:
https://github.com/RaRe-Technologies/gensim/blob/develop/release/annotate_pr.py

Reply:
I was talking about this script: https://github.com/RaRe-Technologies/gensim/blob/develop/release/generate_changelog.py

Reply:
Yes. Once the two scripts output stuff in the same format, the consistency problem will be solved, right?

Reply:
I may be misunderstanding, but why do we need two scripts? `generate_changelog.py` seems to be doing what we need = generate formatted CHANGELOG. When / why would we use `annotate_pr.py`?

Reply:
I'm using annotate_pr.py for each PR (since a few week ago). I'll use generate_changelog.py for the next release. I'll keep whichever ends up being the most convenient - for now, it seems to be the former, but time will tell.